io/net/bonding.py.data/bonding.yaml

```diff
@@ -18,3 +18,5 @@ peer_ip: "85.85.85.85"
 peer_interfaces: "eth3"
 bond_name: "bond13"
 user_name: "root"
+peer_bond_needed: False
+wait_time: "10"
```

Comment:
@hathyaga If it is for peer only, please indicate that. The parameter name can be `peer_wait_time`.
Regarding script taking long time, there are ways to implement this with wait time :-)